### DISCUSSION TITLE
How we can create a .exe setup for the uno application.

### DISCUSSION THREAD

**Discussion by VikasSaxenaxcdify** [unoplatform/uno#8574] [Q&A]
How we can create a .exe setup for the uno application.When we publish the app using "create a web app" option in "package and publish" menu. Its size is too big. Can we provide a framework download from website?

**Comment by jeromelaban** [comment]
Can you specify which application platform you are talking about? Is it the webassembly app?

**Reply by VikasSaxenaxcdify** [reply]
No, I am using the winui3 application.

**Reply by jeromelaban** [reply]
Can you explain the steps you're using to get to "Package and publish"? Can you make a small video and post it here?

**Reply by VikasSaxenaxcdify** [reply]
![7](https://user-images.githubusercontent.com/102783354/165030721-bf54afb2-6ca6-4a17-bb20-132134b78d9c.png)
![1](https://user-images.githubusercontent.com/102783354/165030733-704a2249-9fa2-4cc0-8b07-103455cfe061.png)
![2](https://user-images.githubusercontent.com/102783354/165030741-9d03a955-000a-417b-a53f-60258f128354.png)
![3](https://user-images.githubusercontent.com/102783354/165030747-fb4b7002-8508-4a6e-b0c9-cb449762639a.png)
![4](https://user-images.githubusercontent.com/102783354/165030749-5cfca873-49af-416c-aa0d-8cbb02ed9a73.png)
![5](https://user-images.githubusercontent.com/102783354/165030754-6caa4c39-b4c5-4253-a05d-0aa373954220.png)
![6](https://user-images.githubusercontent.com/102783354/165030758-410ea964-cfc5-4101-9eb7-34f9a96dcc2e.png)

**Reply by jeromelaban** [reply]
Thanks for the update. This is a question for the Windows team: https://github.com/microsoft/WindowsAppSDK/issues/new/choose
